content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -144,9 +144,432 @@ dependencies {
+out [opentelemetry-spring-webflux-5.3](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-webflux/spring-webflux-5.3) to learn more about the
+OpenTelemetry WebClientFilter.
+
+#### Example Trace - WebClient Span
+
+```json
+{
+  "traceId": "0371febbbfa76b2e285a08b53a055d17",
+  "parentId": "9b782243ad7df179",
+  "id": "1b14a2fc89d7a762",
+  "kind": "CLIENT",
+  "name": "http post",
+  "timestamp": 1596841406109125,
+  "duration": 25137,
+  "localEndpoint": {
+    "serviceName": "sample_trace",
+    "ipv4": "XXX.XXX.X.XXX"
+  },
+  "tags": {
+    "http.method": "POST",
+    "http.status_code": "200",
+    "http.url": "/spring-webflux/sample/web-client",
+    "net.peer.name": "localhost",
+    "net.peer.port": "8082"
+  }
+}
+```
+
+## Configuration
+
+### Exporter Configurations
+
+You will need to add the dependency of the exporter you choose
+
+{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>
+    <version>{{% param vers.otel %}}</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-zipkin</artifactId>
+    <version>{{% param vers.otel %}}</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-otlp</artifactId>
+    <version>{{% param vers.otel %}}</version>
+  </dependency>
+</dependencies>
+```
+
+{{% /tab %}} {{% tab header="Gradle (`gradle.build`)" lang=Gradle %}}
+
+```kotlin
+dependencies {
+  implementation("io.opentelemetry:opentelemetry-exporter-jaeger:{{% param vers.otel %}}")
+  implementation("io.opentelemetry:opentelemetry-exporter-zipkin:{{% param vers.otel %}}")
+  implementation("io.opentelemetry:opentelemetry-exporter-otlp:{{% param vers.otel %}}")
+}
+```
+
+{{% /tab %}} {{< /tabpane>}}
+
+This package provides auto configurations for [OTLP](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/otlp), [Jaeger](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/jaeger), [Zipkin](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/zipkin), and [Logging](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging) Span Exporters.
+
+If an exporter is present in the classpath during runtime and a spring bean of the exporter is missing from the spring application context. An exporter bean is initialized and added to a simple span processor in the active tracer provider. Check out the implementation [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java).
+
+#### Enabling/Disabling Exporters
```

Comment:
I would include the _Enabling/Disabling Exporters_ section to a `Spring autoconfiguration` part